modules/ROOT/pages/mobile-developer-console.adoc

```diff
@@ -12,6 +12,8 @@ and your local app in development. This configuration file is used to initialize
 and to connect to the back-end {product-name} you have provisioned on OpenShift.
 
 [#provision-mdc]
+== Provisioning Mobile Developer Console
```

Comment:
@finp it's possible this has changed but from doing the docs during the last release we normally had docs that were duplicated across different sections in partials and they were the `include::`d. The way we are linking out to the content in a different section feels different to the approach we've taken in other areas in the docs. Is it disruptive to the user to have to go another doc to follow and then come back once the section is done to here to follow on with these docs? cc @jhellar

Reply:
let's avoid linking out when possible.